google/oauth2/id_token.py

```diff
@@ -59,6 +59,13 @@
 import json
 import os
 
+try:
+    import jwt as jwt_lib  # type: ignore
+except ImportError as caught_exc:  # pragma: NO COVER
+    raise ImportError(
+        "The pyjwt library is not installed from please install the pyjwt package to use the jwk certs format."
```

Comment:
The message seems to have an extra "from"? Is something supposed to come after the "from"?